cpp/daal/include/sycl/internal/types_utils_cxx11.h

```diff
@@ -163,12 +166,8 @@ class BufferFiller
         template <typename T>
         void operator()(Typelist<T>)
         {
-            auto dst              = dstUnivers.get<T>().toSycl();
-            cl::sycl::event event = queue.submit([&](cl::sycl::handler & cgh) {
-                auto acc = dst.template get_access<cl::sycl::access::mode::write>(cgh);
-                cgh.fill(acc, static_cast<T>(value));
-            });
-            event.wait();
+            auto dst = dstUnivers.get<T>();
+            fillBuffer(dst, dst.size(), static_cast<T>(value));
```

Comment:
I'd rather comment the lines you've deleted because it'd be easier to turn them back when the bug disappears. I suggest creating an internal task to make it.

Reply:
added it as a commented block